app/views/devise/shared/_links.html.erb

```diff
@@ -3,7 +3,7 @@
 <% end -%>
 
 <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
-  <%= link_to "Sign up", new_registration_path(resource_name) %><br />
+  <%= link_to "New to PlaceMe!", new_registration_path(resource_name) %><br />
```

Comment:
Not sure what you mean!
The docs for building links with the `link_to` helper are [here](http://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to).

Normally you just need a string with the text you want to show in the link and the url for the link. In this case if you want to have translated the 'New to PlaceMe!' you need to pass the localized string. Ie `t('.sign_up_btn')`. You can see [here](http://guides.rubyonrails.org/i18n.html#lazy-lookup) how rails is going to look up the translations.